src/nvim/eval.c

```diff
@@ -15509,6 +15451,35 @@ static void f_sort(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   do_sort_uniq(argvars, rettv, true);
 }
 
+/// "stdioopen()" function
+static void f_stdioopen(typval_T *argvars, typval_T *rettv, FunPtr fptr)
```

Comment:
Can't we have e.g. a generic `chanopen()` function, instead of `sockconnect`, `stdioopen`, ...